### PR TITLE
Avoid Type Error in Breadcrumbs

### PR DESCRIPTION
**Pull request by jcq**
## Description
This adjusts the `Breadcrumbs` component so that it doesn't error out in certain routing configurations. It simply adds optional chaining operators to prevent said errors.

## Tech notes
The usage of optional chaining operators required using relevant Babel plugins, which in turn led to upgrading of Babel libraries. This PR now updates all relevant Babel packages and config to match that of main `caseflow` repo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,14 +26,32 @@
...
-    "@department-of-veterans-affairs/eslint-config-appeals": "https://github.com/department-of-veterans-affairs/eslint-config-appeals#f50522e",
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-stage-0": "^6.24.1",
-    "babel-runtime": "^6.26.0",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.11.4",
```
Comment (meckilabritt-navapbc) [inline package.json:30]: Just curious, how were js tests failing with? version 6 and the use of option chaining?

File: config/getWebpackConfig.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = (callingDirname, entry) => {
     entry: _.compact([
       'es5-shim/es5-shim',
       'es5-shim/es5-sham',
-      'babel-polyfill',
+      // 'babel-polyfill',
```
Comment (leikkisa) [inline config/getWebpackConfig.js:14]: Is this commented out because it's deprecated?

File: .nvmrc [hunk 3]
```diff
@@ -1 +1 @@
-6.11.3
+12.13.0
```
Comment (leikkisa) [inline .nvmrc:1]: Such a big jump!

File: components/Breadcrumbs.jsx [hunk 4]
```diff
@@ -8,9 +8,9 @@ import _ from 'lodash';
 // When passed a child component with Route or PageRoute objects that have
 // breadcrumb properties set, this will generate breadcrumbs as links to
 // those other routes.
-const getElementsWithBreadcrumbs = (element) => React.Children.toArray(element.props.children).
+const getElementsWithBreadcrumbs = (element) => React.Children.toArray(element?.props?.children).
   reduce((acc, child) => {
-    if (child.props.breadcrumb) {
+    if (child?.props?.breadcrumb) {
```
Comment (leikkisa) [inline components/Breadcrumbs.jsx:13]: It didn't occur to me before that our new magic ability to do this was from your Babel upgrades! 🥇

**Review by leikkisa — COMMENTED** [review]
I have a question about the commented out line. And I'm curious if this is the sort of thing we'd want to test in UAT, I guess that'd happen through a Caseflow PR.

Other than my question - this looks good to me, thanks for updating dependencies!

**Comment by meckilabritt-navapbc** [comment]
This LGTM. How were the tests failing with the additional of optional chaining?

File: package.json (hunk 1 above)
Comment (jcq) [reply package.json:30]: Jest tests were failing when using optional chaining due to the Babel config here not having been updated to include the plugin that enables the optional chaining feature. The updated plugin for that expected Babel 7.x+, so I just went ahead and upgraded all the things

File: .nvmrc (hunk 3 above)
Comment (jcq) [reply .nvmrc:1]: Oof, yeah. It apparently hadn't caused failures before, but the new version of Babel wouldn't run with such an ancient version of Node

File: config/getWebpackConfig.js (hunk 2 above)
Comment (jcq) [reply config/getWebpackConfig.js:14]: Yup, but leaving it in was an oversight — I'll remove!

**Review by leikkisa — APPROVED** [review]
LGTM

**Pull request merged**